server/migrations/20210429022658-fix-substrate-address.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const { checkAddress, decodeAddress, encodeAddress } = require('@polkadot/util-crypto');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const addresses = await queryInterface.sequelize.query('SELECT id, address, chain FROM "Addresses";');
+    const chains = await queryInterface.sequelize.query('SELECT id, network, base, ss58_prefix FROM "Chains";');
+    const promises = [];
+    const removeAddressIds = [];
+
+    for (const addr of addresses[0]) {
+      const { id, address, chain } = addr;
+      const ch = chains[0].find((_) => _.network === chain);
+      if (ch && ch.base === 'substrate' && !!ch.ss58_prefix) {
```

Comment:
I believe polkadot has an ss58_prefix of 0, so you may want to change the `!!ch.ss58_prefix` expression.

Reply:
Thanks, @jnaviask I will fix this.

Reply:
done